packages/react/src/themes/teams/components/RadioGroup/radioGroupItemStyles.ts

```diff
@@ -31,6 +31,7 @@ const radioStyles: ComponentSlotStylesInput<
 
   icon: ({ props: p, variables: v }): ICSSInJSStyle => ({
     margin: `0 ${pxToRem(10)} 0 0`,
+    padding: 0,
```

Comment:
I am confused with this change... Can you explain what this was added?

Reply:
yes, have addressed related visual regression in icon's styles, thank you!